_content/av/food-of-sinful-demons_barstow-geoff.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Food of Sinful Demons"
+authors:
+  - "Geoff Barstow"
+external_url: "https://podcasts.apple.com/us/podcast/new-books-in-buddhist-studies/id458210899?i=1000451889741"
+source_url: "https://newbooksnetwork.com/geoffrey-barstow-food-of-sinful-demons-meat-vegetarianism-and-the-limits-of-buddhism-in-tibet-columbia-up-2018/"
+course: animals
```

Comment:
🤔  We just had that BSP interview with the same guy, right? Maybe we should only include one of the other in the `animals` tag. Maybe we can move this one to Tibetan?? What do you think?